src/mobile.wiki

```diff
@@ -63,10 +63,16 @@ software.
 
 For all mobile platforms:
 
-* Install {{{npm}}} (on Debian and Ubuntu, {{{sudo apt-get install npm}}}).
+* Install {{{NodeJS}}} and {{{NPM}}} (see
+  [[https://nodejs.org/en/download/package-manager/|the official website]]). If
+  you have some issues with your package manager, you can also use
+  [[https://github.com/creationix/nvm|nvm]].
```

Comment:
@vasilisp

> The Node ecosystem we know and love.

:laughing:

Are you sure NPM suffices? NodeJS must be installed no? I only used once the package manager for Node and I switched to NVM.